### PR TITLE
Add hover effect on the clickable username greeting

### LINKED ISSUE #3473: No hover effect on clickable username greeting
Maybe add an underline when hovering.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/60685084-no-hover-effect-on-clickable-username-greeting?utm_campaign=plugin&utm_content=tracker%2F21853994&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F21853994&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by TPGPL**
- Closes #3473 

*might look a bit weird though*

---

### CODE REVIEW

**Comment by notbakaneko** [comment]
Yeah, we're not doing this because it does look weird

**Comment by notbakaneko** [comment]
no wait, that was something else

File: resources/views/home/user.blade.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
         'title' => trans('home.user.header.welcome', ['username' => link_to(
             route('users.show', Auth::user()),
             Auth::user()->username,
-            ['class' => 'link link--white link--no-underline']
```
Comment (nekodex) [inline resources/views/home/user.blade.php]: `.link--underline` is not a valid class, you probably want to just remove it. `.link--no-underline` is what disables the underline on hover, so removing this class will do what you want.

**Comment by TPGPL** [comment]
Applied

**Pull request merged**